packages/@aws-cdk/aws-events-targets/README.md

```diff
@@ -103,9 +103,9 @@ declare const logGroup: logs.LogGroup;
 declare const rule: events.Rule;
 
 rule.addTarget(new targets.CloudWatchLogGroup(logGroup, {
-  logEvent: targets.LogGroupTargetInput({
-    timestamp: events.EventField.from('$.time'),
-    message: events.EventField.from('$.detail-type'),
+  logEvent: targets.LogGroupTargetInput.fromObject({
```

Comment:
Heads up to reviewers: these updates that aren't specifically related to ECS were required because they would not compile when running Rosetta.